lib/routes.js

```diff
@@ -88,22 +89,22 @@ exports.routes.push({
 
 exports.routes.push({
     method: 'GET',
-    path: '/tutorials/{tutorial?}',
+    path: '/tutorials/{lang}/{tutorial?}',
```

Comment:
Maybe its best `/tutorials/tutorial/lang` and default to english when the `/lang` its not avaliable. That way the links in the internet won't stop working.